PostPairsTrading.md

```diff
@@ -227,33 +220,19 @@ This precisely meets one of our objectives: getting **a comprehensive method for
 
 ## Real-time spread calculation
 
-Now that we have selected a pair of cointegrated indices and understand how to calculate their relationships, let's see how we can create a real-time pair trading scenario.
-
-To do this, we need to focus on the Real-Time Pair Trading Subscriber (RPT). As can be seen in the diagram, this component receives 3 inputs.
-
-Let's start with the easy part: first, we receive which pair of indexes we have chosen in the previous step (this step, in our case, is done AD-HOC since, as we mentioned, we have chosen these two indices not only because of their cointegration but also because they are indices whose tick data is public).
-
-The next step is to receive the data for this pair of indices. For this, we need to subscribe to the tickerplant using the tick architecture function `.u.sub`. Here, we need to specify which table and which symbols (indexes in our case) we want to subscribe to. This information needs to be passed through the connection to the tickerplant. 
-
-> 💡 `.z.x` in the KDB+/Q language is an environment variable that stores the command-line arguments passed to a q script.
+Now that we have selected a pair of cointegrated indices and built a model to calculate their relationships, it's time to formalize its subscription as a real-time component. Once we start receiving data from the TP, we can apply the model to produce the spreads, which will then be sent to the dashboard.
+![Arch-bottom](resources/general-architecture-bottom.png)
 
+Real-time components can manifest their interest for a particular table and for a subset of symbols. As a result from previous steps, we know we are interested on the quotes for SP500 and NASDAQ100:

```

Comment:
We should briefly explain why those two hardcoded symbols appear (``` `SP500`NASDAQ100 ```), shouldn't we? I mean, perhaps the reader misses something more automated, right?

Reply:
Yes, this is one of the problems that I feel while connecting pieces. They seem like independent things while DFA and RPT are completely tied. What we do is weird and difficult to justify, but I'm not sure if we could change it easily. Let me give it a try.